### DISCUSSION TITLE
Different test result

### DISCUSSION THREAD

**Discussion by seohyeon0322** [ANLAB-KAIST/KENSv3#79] [Project Q&A 2. Transport Layer (TCP)]
I am using macbook and my teammate is using Window + WSL. However, when we test our code with unreliable test case, the result is different. How should we do? Does the result can be differ according to the environment? Then, what would be the environment that will be used when grading?

**Comment by Pusnow** [comment]
The unreliable test cases have non-determinism due to their randomness (bit flips).

You should pass test cases in both environments.

Also, we will use 5 different `RANDOM_SEED`s.
To get full credit, you should use various `RANDOM_SEED`s.

**Reply by seohyeon0322** [reply]
Thank you!
